mockk/src/main/kotlin/io/mockk/impl/annotations/JvmMockInitializer.kt

```diff
@@ -55,6 +58,17 @@ class JvmMockInitializer(val gateway: MockKGateway) : MockKGateway.MockInitializ
                     arrayOf()
                 )
             }
+
+            if (property.findAnnotation<InjectMockKs>() != null) {
+                val injectMockProperty = property as KProperty1<Any, Any>
```

Comment:
Better keep it semantically named alike `propertyToInjectMockKs`. OR as an option in small scoped variables I use simpler name alike just `property`

Reply:
Are you sure that it is only KProperty1? KProperty2 is probably a property with extension or something alike. Please check this case, as it can result in class cast